front/src/applications/stdcm/utils/formatSimulationReportSheet.ts

```diff
@@ -142,3 +142,39 @@ export function getOperationalPointsWithTimes(
 
   return opResults;
 }
+
+/**
+ * Converts a base64 image into a JPEG blob while reducing quality.
+ * @param base64Image - Image in base64
+ * @param quality - Image quality (between 0 and 1, where 1 is the best quality)
+ * @returns {Promise<Blob | null>} - Return an optimised JPEG Blob
+ */
+export const base64ToJpeg = (base64Image: string, quality: number): Promise<Blob | null> =>
```

Comment:
Nit: `Blob | null` can be replaced with `Blob` only since the promise is rejected on error.